.rollup.js

```diff
@@ -20,9 +20,10 @@ const output = isCLI
 	{ file: 'index.mjs', format: 'esm', sourcemap: true }
 ];
 
-const targets = isCLI || isPostCSS || !isBrowser ? { node: 8 } : 'last 2 versions, not dead';
+const targets = isCLI || isPostCSS || !isBrowser ? { node: 10 } : 'last 2 versions, not dead';
```

Comment:
We can update this to support Node v12. Node v10 is no longer supported, even for maintenance.

See: https://nodejs.org/en/about/releases/

```suggestion
const targets = isCLI || isPostCSS || !isBrowser ? { node: 12 } : 'last 2 versions, not dead';
```